demo/video_demo.py

```diff
@@ -19,8 +19,8 @@ def parse_args():
     parser.add_argument('--out', type=str, help='Output video file')
     parser.add_argument('--show', action='store_true', help='Show video')
     parser.add_argument(
-        '--wait-time',
-        type=float,
+        '--wait_time',
```

Comment:
All args in mmdet use `-` instead of `_`. It will be converted automatically when parsing args.
```suggestion
        '--wait-time',
```